packages/x-components/src/design-system/components/sliding-panel/default.scss

```diff
@@ -1,4 +1,7 @@
 .x-sliding-panel {
```

Comment:
There are a few pointer-events: all that you can now remove, both in `sliding-panel.vue` and `sliding-panel/default.scss`

Reply:
We must keep them, the buttons only change the opacity, and we have to avoid the click when opacity is 0

Reply:
Yep, my bad, didn't realise about this.